doc/faq.rst

```diff
@@ -4,45 +4,29 @@ Frequently Asked Questions
-
-Sometimes, we really want to work with collections of higher dimensional arrays
+pandas is a fantastic library for analysis of low-dimensional labelled data -
+if it can be sensibly described as "rows and columns", pandas is probably the
+right choice.  However, sometimes we want to use higher dimensional arrays
 (`ndim > 2`), or arrays for which the order of dimensions (e.g., columns vs
 rows) shouldn't really matter. For example, climate and weather data is often
 natively expressed in 4 or more dimensions: time, x, y and z.
 
-Pandas does support `N-dimensional panels`__, but the implementation
-is very limited:
-
-__ http://pandas.pydata.org/pandas-docs/stable/dsintro.html#panelnd-experimental
-
-  - You need to create a new factory type for each dimensionality.
-  - You can't do math between NDPanels with different dimensionality.
-  - Each dimension in a NDPanel has a name (e.g., 'labels', 'items',
-    'major_axis', etc.) but the dimension names refer to order, not their
-    meaning. You can't specify an operation as to be applied along the "time"
-    axis.
-
-Fundamentally, the N-dimensional panel is limited by its context in pandas's
-tabular model, which treats a 2D ``DataFrame`` as a collections of 1D
-``Series``, a 3D ``Panel`` as a collection of 2D ``DataFrame``, and so on. In
-my experience, it usually easier to work with a DataFrame with a hierarchical
-index rather than to use higher dimensional (*N > 3*) data structures in
-pandas.
-
-Another use case is handling collections of arrays with different numbers of
-dimensions. For example, suppose you have a 2D array and a handful of
-associated 1D arrays that share one of the same axes. Storing these in one
-pandas object is possible but awkward -- you can either upcast all the 1D
-arrays to 2D and store everything in a ``Panel``, or put everything in a
-``DataFrame``, where the first few columns have a different meaning than the
-other columns. In contrast, this sort of data structure fits very naturally in
-an xarray ``Dataset``.
+Pandas has historically supported N-dimensional panels, but deprecated them in
+version 0.20 in favor of Xarray data structures.  There are now built-in methods
+on both sides to convert between pandas and Xarray, allowing for more focussed
+development effort.  Xarray objects have a much richer model of dimensionality -
+if you were using Panels:
+
+- You need to create a new factory type for each dimensionality.
+- You can't do math between NDPanels with different dimensionality.
+- Each dimension in a NDPanel has a name (e.g., 'labels', 'items',
+  'major_axis', etc.) but the dimension names refer to order, not their
+  meaning. You can't specify an operation as to be applied along the "time"
+  axis.
+- You often have to manually convert collections of pandas arrays
+  (Series, DataFrames, etc) to have the same number of dimensions.
+  In contrast, this sort of data structure fits very naturally in an
+  xarray ``Dataset``.
```

Comment:
This would be a great place to link with our doc section on transitioning from Panel to xarray:
http://xarray.pydata.org/en/stable/pandas.html#transitioning-from-pandas-panel-to-xarray

Reply:
Done (just below).